hazelcast-spring/src/main/java/com/hazelcast/spring/AbstractHazelcastBeanDefinitionParser.java

```diff
@@ -398,7 +398,7 @@ protected BeanDefinition getEvictionConfig(final Node node) {
             }
 
             try {
-                EvictionConfigHelper.checkEvictionConfig(evictionPolicyValue, comparatorClassNameValue, comparatorBean);
+                checkEvictionConfig(evictionPolicyValue, comparatorClassNameValue, comparatorBean, false);
```

Comment:
I know what you mean, but the obvious decision path for the `EvictionConfig` check is "is it a config for Near Cache or (`IMap`|`ICache`|I think there was a third one)". So there is not really a nice way to describe the "not a Near Cache". This is why we have this `false` parameter in this one place.

I could make two wrapper methods like `checkEvictionConfigForNearCache()` and `checkEvictionConfigForNonNearCache()`, but that looks a bit too much for the very few calls we have for those methods.